src/cfn/template.yaml

```
@@ -776,7 +791,7 @@ Resources:
       FunctionName: !Sub RekogDemoSetup${CollectionId}
       Runtime: nodejs14.x
       Handler: index.handler
-      CodeUri: ../backend/functions/setup/
+      CodeUri: ../functions/setup/
```

```suggestion
      CodeUri: ../backend/functions/setup/
```

I think this change will break the deploy links. If you check the `build` task in the `package.json`, that does some stuff including:
1. packaging the lambdas and save them into a gitignored `backend` folder
2. cfn-flip the template and save to a gitignored `templates` folder

And that is how the things will be also zipped during the deploy. if you are deploying locally, run the `npm run build` and then deploy the `templates/template.yaml` and I think this should work as it was.

---

Yeah got ya, was wondering what have I missed as cfn was breaking. Will revert, Thanks!